testutil/singlevm/verify.sh

```diff
@@ -281,32 +283,6 @@ checkEventStatus $event_counter "Unmapped"
 
 "$ciao_gobin"/ciao-cli external-ip list
 
-#Test for External IP Failures
-
-#Map failure
```

Comment:
@kaccardi @markdryan this was a corner case that  I wanted to validate. However till we have CNCI ssh key provisioning support, we can disable the same. Without this case one of the events (which in a sane ciao setup, should never occur will not get validated).